src/Helpers/SentryHelper.php

```diff
@@ -0,0 +1,61 @@
+<?php
+
+declare(strict_types=1);
+
+namespace Baldinof\RoadRunnerBundle\Helpers;
+
+use Jean85\PrettyVersions;
+use Sentry\ClientInterface;
+use Sentry\Event;
+use Sentry\FlushableClientInterface;
+use Sentry\UserDataBag;
+
+class SentryHelper
+{
+    public static function isVersion3(): bool
+    {
+        $version = PrettyVersions::getVersion('sentry/sentry')->getPrettyVersion();
+
+        return version_compare($version, '3.0.0', '>=');
```

Comment:
The result of this can be cached in a static property.

Reply:
done!